.werft/build.ts

```diff
@@ -289,6 +314,254 @@ interface DeploymentConfig {
+        werft.done(installerSlices.INSTALLER_INIT);
+    } catch (err) {
+        werft.fail(installerSlices.INSTALLER_INIT, err)
+    }
+
+    // prepare a proper config file
+    try {
+        werft.log(installerSlices.INSTALLER_RENDER, "Post process the base installer config file and render k8s manifests");
+        const PROJECT_NAME="gitpod-core-dev";
+        const CONTAINER_REGISTRY_URL=`eu.gcr.io/${PROJECT_NAME}/build/`;
+        const CONTAINERD_RUNTIME_DIR = "/var/lib/containerd/io.containerd.runtime.v2.task/k8s.io";
+
+        // get some values we need to customize the config and write them to file
+        exec(`yq r ./.werft/values.dev.yaml components.server.blockNewUsers \
+        | yq prefix - 'blockNewUsers' > ./blockNewUsers`, { slice: installerSlices.INSTALLER_RENDER });
+        exec(`yq r ./.werft/values.variant.cpuLimits.yaml workspaceSizing | yq prefix - 'workspace' > ./workspaceSizing`, { slice: installerSlices.INSTALLER_RENDER });
+
+        // merge values from files
+        exec(`yq m -i --overwrite config.yaml ./blockNewUsers`, { slice: installerSlices.INSTALLER_RENDER });
+        exec(`yq m -i config.yaml ./workspaceSizing`, { slice: installerSlices.INSTALLER_RENDER });
+
+        // write some values inline
+        exec(`yq w -i config.yaml certificate.name ${PROXY_SECRET_NAME}`, {slice: installerSlices.INSTALLER_RENDER});
+        exec(`yq w -i config.yaml containerRegistry.inCluster false`, {slice: installerSlices.INSTALLER_RENDER});
+        exec(`yq w -i config.yaml containerRegistry.external.url ${CONTAINER_REGISTRY_URL}`, {slice: installerSlices.INSTALLER_RENDER});
+        exec(`yq w -i config.yaml containerRegistry.external.certificate.kind secret`, {slice: installerSlices.INSTALLER_RENDER});
+        exec(`yq w -i config.yaml containerRegistry.external.certificate.name ${IMAGE_PULL_SECRET_NAME}`, {slice: installerSlices.INSTALLER_RENDER});
+        exec(`yq w -i config.yaml domain ${deploymentConfig.domain}`, {slice: installerSlices.INSTALLER_RENDER});
+        exec(`yq w -i config.yaml jaegerOperator.inCluster false`, {slice: installerSlices.INSTALLER_RENDER});
+        exec(`yq w -i config.yaml workspace.runtime.containerdRuntimeDir ${CONTAINERD_RUNTIME_DIR}`, {slice: installerSlices.INSTALLER_RENDER});
+
+        if ((deploymentConfig.analytics || "").startsWith("segment|")) {
+            exec(`yq w -i config.yaml analytics.writer segment`, {slice: installerSlices.INSTALLER_RENDER});
+            exec(`yq w -i config.yaml analytics.segmentKey ${deploymentConfig.analytics!.substring("segment|".length)}`, {slice: installerSlices.INSTALLER_RENDER});
+        } else if (!!deploymentConfig.analytics) {
+            exec(`yq w -i config.yaml analytics.writer ${deploymentConfig.analytics!}`, {slice: installerSlices.INSTALLER_RENDER});
+        }
+
+        if (withObservability) {
+            // TODO: there's likely more to do...
```

Comment:
It would need to deploy the observability stack. Before deploying Gitpod. We'll take that in a separate PR.

Reply:
Thank you for the heads up! If you want help testing let me know?

Reply:
Will do :)